testsuite/forge.py

```diff
@@ -37,6 +38,17 @@
 from forge_wrapper_core.time import SystemTime, Time
 from forge_wrapper_core.cluster import Cloud, ForgeCluster, ForgeJob
 
+log = logging.getLogger("")
```

Comment:
put this in a function and call that from either main or a decorator on commands